src/test/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/AlkylStructureFragmenterTest.java

```diff
@@ -0,0 +1,754 @@
+     * This test focuses on extraction of a singular ring and separation of linear, conjugated Pi systems from side chains,
+     * tested with a derivative of the molecule Dehydropinguisanin after fragmentation with the (in MORTAR included) ErtlFunctionalGroupsFinder.
+     *
+     * @throws InvalidSmilesException if SMILES is not correctly parsed of otherwise faulty
+     * @throws CloneNotSupportedException if something goes wrong during cloning step in fragmentation
+     */
+    @Test
+    public void specificTest03() throws InvalidSmilesException, CloneNotSupportedException {
+        //test structure: C=CC=CCC1(C)C(C)CCC1C (Dehydropinguisanin)
+        SmilesParser tmpParser = new SmilesParser(SilentChemObjectBuilder.getInstance());
+        IAtomContainer tmpTestStructureAC = tmpParser.parseSmiles("C=CC=CCC1(C)C(C)CCC1C");
+        AlkylStructureFragmenter tmpASF = this.getDefaultASFInstance(tmpTestStructureAC, false,
+                false, true);
+        List<String> tmpFragmentsACList = this.generateSMILESFromACList(tmpASF.fragmentMolecule(tmpTestStructureAC));
+        List<String> tmpExpectedSMILESList = new ArrayList<>();
+        tmpExpectedSMILESList.add("C=CC=C");
+        tmpExpectedSMILESList.add("C");
+        tmpExpectedSMILESList.add("C");
+        tmpExpectedSMILESList.add("C");
+        tmpExpectedSMILESList.add("C");
+        tmpExpectedSMILESList.add("C1CCCC1");
+        Assertions.assertTrue(this.compareListsIgnoringOrder(new ArrayList<>(tmpFragmentsACList),
+                new ArrayList<>(tmpExpectedSMILESList)));
+    }
+    /**
+     * Test for correct deepCopy methods by copying a butene molecule which used to make problems in earlier versions.
+     *
+     * @throws FileNotFoundException if test structures file can not be found or accessed
+     * @throws URISyntaxException if syntax of path to test structures file is wrong
+     */
+    @Test
+    public void deepCopyButeneTest() throws FileNotFoundException, URISyntaxException {
+        AlkylStructureFragmenter tmpASF = new AlkylStructureFragmenter();
+        tmpASF.setFragmentSaturationSetting(IMoleculeFragmenter.FRAGMENT_SATURATION_OPTION_DEFAULT);
+        tmpASF.setKeepNonFragmentableMoleculesSetting(AlkylStructureFragmenter.KEEP_NON_FRAGMENTABLE_MOLECULES_SETTING_DEFAULT);
+        tmpASF.setFragmentSideChainsSetting(AlkylStructureFragmenter.FRAGMENT_SIDE_CHAINS_SETTING_DEFAULT);
+        tmpASF.setMaxChainLengthSetting(AlkylStructureFragmenter.MAX_CHAIN_LENGTH_SETTING_DEFAULT);
+        tmpASF.setIsolateQuatCarbonsSetting(AlkylStructureFragmenter.ISOLATE_TERT_QUAT_CARBONS_SETTING_DEFAULT);
+        tmpASF.setSeparateTertQuatCarbonFromRingSetting(AlkylStructureFragmenter.SEPARATE_TERT_QUAT_CARBON_FROM_RING_SETTING_DEFAULT);
+        IAtomContainerSet tmpReadAtomContainerSet = this.readStructuresToACSet("de.unijena.cheminf.mortar.model.fragmentation.algorithm.ASF/ASF_Butene_Test.mol");
```

Comment:
It should not be necessary to give the full package path for the resource, just the file name should be sufficient.